acctest/doc.go

```diff
@@ -0,0 +1,31 @@
+package acctest
+
+// Package acctest provides the ability to opt-in to the new binary test driver. The binary
+// test driver allows you to run your acceptance tests with a binary of Terraform instead of
+// a source code version included with the testing code. This allows for a number of important
+// enhancements, but most notably a more realistic testing experience and matrix testing
+// against multiple versions of Terraform CLI. This also allows the SDK to be completely
+// separated, at a dependency level, from the Terraform CLI.
```

Comment:
```suggestion
// separated, at a dependency level, from the Terraform CLI, as long as it is >= 0.12.0
```
The binary driver needs to be API compatible with those json commands.